docs/deployment/local-codebase.md

```diff
@@ -8,7 +8,6 @@ Update the `vagrant_synced_folders` configuration to sync your local Drupal code
 vagrant_synced_folders:
   - local_path: ~/Sites/my-drupal-site
     destination: /var/www/my-drupal-site
-    id: drupal
```

Comment:
@geerlingguy noticed this change when updating locally. We shouldn't be specifying an ID per site anymore when creating multiple sites using one drupalvm instance?